sdk/rust/src/lib.rs

```diff
@@ -95,12 +95,8 @@ pub mod pg;
 /// Implementation of the Spin MySQL database interface.
 pub mod mysql;
 
-/// Implementation of the spin config interface.
-#[allow(missing_docs)]
-pub mod config {
-    /// Exports the generated Spin config items.
-    pub use super::wit::v1::config::{get_config as get, Error};
-}
+/// Implementation of the spin variables interface.
```

Comment:
Can we do `#[doc(inline)]` instead and use doc comments on the variables interface to populate this?